python/doc/examples/data_analysis/distribution_fitting/plot_estimate_gev_fremantle.py

```diff
@@ -0,0 +1,166 @@
+result1 = factory.buildMethodOfLikelihoodMaximizationEstimator(sample)
+
+# %%
+# Print the estimated parameter values
+estimate = result1.getDistribution()
+desc = estimate.getParameterDescription()
+p = estimate.getParameter()
+print(", ".join([f"{param}: {value:.3f}" for param, value in zip(desc, p)]))
+
+# %%
+# Inspect the estimated Gaussian parameter distribution
+parameterEstimate = result1.getParameterDistribution()
+print(parameterEstimate)
+
+# %%
+# Inspect the covariance matrix
+V = parameterEstimate.getCovariance()
+print(V)
+
+# %%
+# Inspect the standard deviation
+stddev = parameterEstimate.getStandardDeviation()
+print(stddev)
+
+# %%
+# Extract parameters confidence intervals
+for i in range(3):
+    ci = parameterEstimate.getMarginal(i).computeBilateralConfidenceInterval(0.95)
+    print(desc[i] + ":", ci)
+
+# %%
+# Validate the inference result thanks to some diagnostic plots
+validation = otexp.GeneralizedExtremeValueValidation(result1, sample)
+graph = validation.drawDiagnosticPlot()
+view = otv.View(graph)
+
+# %%
+# Now estimate the parameters with the profile likelihood
+result2 = factory.buildMethodOfProfileLikelihoodMaximizationEstimator(sample)
+
+# %%
+# We can see the confidence interval of :math:`\xi` is a bit smaller with this method
+result2.setConfidenceLevel(0.95)
+print(result2.getParameterConfidenceInterval())
+
+# %%
+# We can also plot the profile likelihood and the confidence interval
+view = otv.View(result2.drawProfileLikelihoodFunction())
+
+# %%
+# Estimate the 10 and 100 years return levels from the MLE estimator
+zm10 = factory.buildReturnLevelEstimator(result1, 10.0)
+return_level10 = zm10.getMean()
+print(f"10 years return level={return_level10}")
+return_level_ci10 = zm10.computeBilateralConfidenceInterval(0.95)
+print(f"CI={return_level_ci10}")
+
+zm100 = factory.buildReturnLevelEstimator(result1, 100.0)
+return_level100 = zm100.getMean()
+print(f"100 years return level={return_level100}")
+return_level_ci100 = zm100.computeBilateralConfidenceInterval(0.95)
+print(f"CI={return_level_ci100}")
+
+# %%
+# Now estimate the 10 years return level with profile likelihood
+result_rl10_prof = factory.buildReturnLevelProfileLikelihoodEstimator(sample, 10.0)
+zm = result_rl10_prof.getParameter()
+print(f"10 years return level (profile)={zm}")
+
+# %%
+# We can see the confidence interval of :math:`z_m` is a bit smaller with this method
+result_rl10_prof.setConfidenceLevel(0.95)
+return_level_ci10 = result_rl10_prof.getParameterConfidenceInterval()
+print(f"CI={return_level_ci10}")
+
+# %%
+# We can also plot the profile likelihood and the confidence interval
+view = otv.View(result_rl10_prof.drawProfileLikelihoodFunction())
+
+# %%
+# Now formulate a non-stationary hypothesis for our parameters
+# First we need the grid of time values (in years)
+mesh = ot.Mesh(data[:, 0])
+
+# %%
+# Now define the basis for each parameter
+# lets say mu is linear with time, the other parameters remaining constant
```

Comment:
This reads more like cliff notes than sentences.

Reply:
You don't need to answer in this PR if you remove the file like I suggested in the introductory paragraph.